stacks/kafka-flink/README.rst

```diff
@@ -41,7 +41,8 @@ It will outline how to acquire and publish data to Kafka, process it using
 Flink, and store the data stream into CrateDB.
 
 The system is part of the CrateDB reference architecture and can be used as a
-blueprint for building own applications.
+blueprint for building own applications, both using a `CrateDB`_ instance
+running on your machine, and using `CrateDB Cloud`_.
```

Comment:
```suggestion
running on your machine, or using `CrateDB Cloud`_.
```

Reply:
This slipped in by accident, it got removed now.